bokeh/util/dependencies.py

```diff
@@ -45,7 +46,7 @@
 # General API
 #-----------------------------------------------------------------------------
 
-def import_optional(mod_name):
+def import_optional(mod_name:str) -> Optional[ModuleType]:
```

Comment:
@mborus, please keep whitespace consistent here in a few other locations (one space after `:`).